x-pack/plugins/apm/public/components/app/TransactionDetails/index.tsx

```diff
@@ -50,18 +50,20 @@ export function TransactionDetails({
   const { urlParams } = useUrlParams();
   const history = useHistory();
   const {
-    data: distributionData,
-    status: distributionStatus,
-  } = useTransactionDistribution(urlParams);
+    distributionData,
+    distributionStatus,
+  } = useTransactionDistributionFetcher();
```

Comment:
The hook now returns `distributionData` and `distributionStatus` instead of `data` and `status`